pages/linux/pacman-sync.md

```diff
@@ -0,0 +1,36 @@
+# pacman --sync
+
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- Install a new package:
+
+`sudo pacman --sync {{package_name}}`
+
+- Synchronize and update all packages (add `--downloadonly` to download only):
```

Comment:
How about this?  Coming back to this page after a while, I read it as trying to add the string `--downloadonly` to the string "download only" (???)
```suggestion
- Synchronize and update all packages (add `--downloadonly` to download the packages and not update them):
```